protocol/0053-PERP-product_builtin_perpetual_future.md

```diff
@@ -364,3 +364,5 @@ Assert that the scaling factor is applied before the funding cap is applied (<a
 Assert all funding payments are correct when a perpetual market is suspended and then terminated via governance. (<a name="0053-PERP-041" href="#0053-PERP-041">0053-PERP-041</a>).
 
 The upper and lower clamp values are being correctly validated as per the [parameters defined in the spec](https://github.com/vegaprotocol/specs/blob/palazzo/protocol/0053-PERP-product_builtin_perpetual_future.md#1-product-parameters). (<a name="0053-PERP-042" href="#0053-PERP-042">0053-PERP-042</a>).
+
+Launch a perpetual futures market which uses the "Last Traded Price" for the "vega side price" for funding calculation. Submit a market update proposal to change this to a composite price with a configuration which uses the impact notional price from the order book. Observe that the new methodology for funding calculations is applied correctly from enactment onwards. (<a name="0053-PERP-043" href="#0053-PERP-043">0053-PERP-043</a>).
```

Comment:
Given its possible for `internalCompositePrice` to be set to `nil`. Can we have two more AC's one that takes a perptual market where it is set to `nil` in its original proposal, and updates it to have a a composite-price configuration, and another that takes a market that was original proposed with a composite-price-configuration and updates it to be set to `nil`?